benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherSelectCandidatesSmallEntryCountBenchmark.cs

```diff
@@ -90,9 +90,9 @@ public void Dfa()
             Span<PathSegment> segments = stackalloc PathSegment[FastPathTokenizer.DefaultSegmentCount];
             var count = FastPathTokenizer.Tokenize(path, segments);
 
-            var candidates = _dfa.SelectCandidates(path, segments.Slice(0, count));
+            var candidates = _dfa.FindCandidates(httpContext, path, segments.Slice(0, count));
```

Comment:
I renamed this method because it's not super clear. Lots of things were using the term `Select` for different tasks.

Reply:
Ok, I'm going to go with `FindCandidateSet` - that will be a little more distinct.